sql/src/main/java/io/crate/analyze/AnalyzedUpdateStatement.java

```diff
@@ -59,8 +69,14 @@ public Symbol query() {
         return query;
     }
 
-    public Multimap<String, Symbol> returningClause() {
-        return returningClause;
+    @Nullable
+    public List<Field> fields() {
+        return fields == null ? null : fields.asList();
```

Comment:
This could probably go into the constructor? Maybe even change the dependencies to `List<Field>` ("don't be greedy"). Maybe also add a short javadoc explaining that fields/returnValues are for the RETURNING clause. 

Also: What's the difference between the returnValues and the fields? Both are symbols.